scene/main/node.cpp

```diff
@@ -3843,6 +3873,7 @@ void Node::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("child_order_changed"));
 	ADD_SIGNAL(MethodInfo("replacing_by", PropertyInfo(Variant::OBJECT, "node", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT, "Node")));
 	ADD_SIGNAL(MethodInfo("editor_description_changed", PropertyInfo(Variant::OBJECT, "node", PROPERTY_HINT_NONE, "", PROPERTY_USAGE_DEFAULT, "Node")));
+	ADD_SIGNAL(MethodInfo("editor_state_changed"));
```

Comment:
We could probably deprecate `editor_description_changed` now.

Reply:
I do something different for that, but it could indeed just be removed. Is it part of our public API? If it is then removing it would possibly break API, and prevent backporting to 4.3?

Reply:
It's exposed, but it's editor-only thing which I assume is rarely used.
It's better to deprecate it in a follow-up though.